### DISCUSSION TITLE
Build for different versions of Ubuntu

### DISCUSSION THREAD

**Discussion by artur-bunko-rnm** [tauri-apps/tauri#10916] [Q&A]
How can I create a build on Ubuntu 24 so that it works on Ubuntu 20 as well?
Do I need to use Tauri 1 for Ubuntu 20 and Tauri 2 for Ubuntu 24?

**Comment by FabianLars** [comment]
On Linux you generally need to build on the oldest system you want the app to still run. An app built on Ubuntu 18 will normally still work on 24 assuming the app is self-contained, or all dependencies are still available.

So in your case, if you build your app on 20.04 and _only_ distribute the AppImage (which is self contained) it will work on 22 and 24.
If you distribute the .deb file then you need one build on 20.04 (or below) for 20 and 22, and another version built on 24 for 24 and above because 24 does not have webkitgtk 4.0 (tauri v1) anymore
